pout/value.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import os
 import traceback
-from collections import KeysView
+from collections.abc import KeysView
```

Comment:
Nice catch, this should actually be deleted here and the import should be moved into the `compat.py` module under both the python 2 import and the python 3 import from the collections module